IrisSoftware/main.py

```diff
@@ -70,7 +70,7 @@ def getFrameWithEyeCoords(self) -> ndarray:
         for (x, y) in eyeCoords:
```

Comment:
We can safely remove drawing the circles since we only care about the eye coords here.